schematics/table/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.ts

```diff
@@ -0,0 +1,131 @@
+import { Component, OnInit, ViewChild, <% if(!!viewEncapsulation) { %>, ViewEncapsulation<% }%><% if(changeDetection !== 'Default') { %>, ChangeDetectionStrategy<% }%> } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { map } from 'rxjs/operators/map';
+import { MatPaginator, MatSort } from '@angular/material';
+import { DataSource } from '@angular/cdk/collections';
+import { of } from 'rxjs/observable/of';
+import { merge } from 'rxjs/observable/merge';
+
+/** Todo: Replace this with your own data model type */
+export interface DataItem {
+  name: string;
+  id: number;
+}
+
+function compare(a, b, isAsc) {
```

Comment:
Can this name be more specific? `sortComparator`? Should also have a comment like 
```ts
/** Simple sort comparator for example ID/Name columns. */
```

I would also move this to the bottom of the file.